css/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+    background-color: #eee;
+    color: #333;
+    font-size: 1.3rem;
+    line-height: 1.5;
+}
+h1, h2{
+    padding: 0.5rem 0;
+}
+p{
+    padding: 0.3rem 0;
+}
+
+#nav-bar{
+    padding: 2rem;
+    display: flex;
+    justify-content: space-between;
+
+}
+ul{
+    display: flex;
+}
+li{
+    list-style-type: none;
+}
```

Comment:
Your code can look cleaner if you apply the linters for formatting your files. :raised_hands: